plugins/modules/provision_workflow_manager.py

```diff
@@ -716,26 +708,27 @@ def get_wired_params(self):
           paramters and stores it for further processing and calling the
           parameters in other APIs.
         """
+        self.log("self.validated_config")
```

Comment:
Instead:
What is the log level? Can you mention the log level?
self.log("self.validated_config: {}".format(self.validated_config))

Reply:
it was for the self reference will delete it